sample/Kmd.Logic.CitizenDocuments.Client.Sample/Kmd.Logic.CitizenDocuments.Client.Sample.csproj

```diff
@@ -17,6 +17,10 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.UserSecrets" Version="3.1.0" />
     <PackageReference Include="Serilog" Version="2.9.0" />
     <PackageReference Include="Serilog.Sinks.Console" Version="3.1.1" />
+    <PackageReference Include="StyleCop.Analyzers" Version="1.1.118">
```

Comment:
Won’t this be inherited by the root directory.build.props file?

Reply:
Was not sure before, but now removed from citizen document sample and client project. 👍